models/gpg_key_commit_verification.go

```diff
@@ -357,6 +425,21 @@ func hashAndVerifyWithSubKeys(sig *packet.Signature, payload string, k *GPGKey)
 	return nil, nil
 }
 
+func verifySSHCommitVerification(sig, payload string, k *PublicKey, committer, signer *User, email string) *CommitVerification {
+	if err := sshsig.Verify(bytes.NewBuffer([]byte(payload)), []byte(sig), []byte(k.Content)); err != nil {
```

Comment:
I'm lost here - when there is a error(`err != nil`) the signature is then verified? I assume you meant this to be the other way around.

Reply:
indeed